test/util/aleTests/ALEPendlePTUSDe.t.sol

```diff
@@ -0,0 +1,638 @@
+pragma solidity ^0.8.13;
+
+import {ICurvePool} from "src/interfaces/ICurvePool.sol";
+import {PendlePTHelper} from "src/util/PendlePTHelper.sol";
+import "test/marketForkTests/PendlePTUSDeMarketForkTest.t.sol";
+import {console} from "forge-std/console.sol";
+import {IMultiMarketTransformHelper} from "src/interfaces/IMultiMarketTransformHelper.sol";
+import {ALE} from "src/util/ALE.sol";
+import {SimpleERC20Escrow} from "src/escrows/SimpleERC20Escrow.sol";
+
+contract ALEPendlePTUSDeTest is PendlePTUSDeMarketForkTest {
+    ALE ale;
+    address userPk = vm.addr(1);
+    PendlePTHelper helper;
+    address userPkEscrow;
+
+    address pendleRouter = address(0x888888888889758F76e7103c6CbF23ABbF58F946);
+    address pendleYT = address(0x96512230bF0Fa4E20Cf02C3e8A7d983132cd2b9F);
+    address marketPT = address(0xcDd26Eb5EB2Ce0f203a84553853667aE69Ca29Ce);
+
+    // Pendle Router Mock
+    address pendleYTHolder =
+        address(0x9844c3688dAaA98De18fBe52499A6B152236896b);
+
+    // Get initial PT
+    // Swap 100K DOLA for PT (recipient helper)
+    bytes pendleDataInitial =
```

Comment:
MY EYES
![image](https://github.com/user-attachments/assets/fedce57a-ce53-4494-a704-cf92a90f48a2)

Reply:
I will just assume this is good and correct